RecoTracker/MeasurementDet/plugins/Chi2ChargeMeasurementEstimatorESProducer.cc

```diff
@@ -102,32 +103,36 @@ namespace {
 
 class  Chi2ChargeMeasurementEstimatorESProducer: public edm::ESProducer{
  public:
+  // static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
   Chi2ChargeMeasurementEstimatorESProducer(const edm::ParameterSet & p);
   virtual ~Chi2ChargeMeasurementEstimatorESProducer(); 
   boost::shared_ptr<Chi2MeasurementEstimatorBase> produce(const TrackingComponentsRecord &);
  private:
-  boost::shared_ptr<Chi2MeasurementEstimatorBase> _estimator;
+  boost::shared_ptr<Chi2MeasurementEstimatorBase> m_estimator;
+  const edm::ParameterSet m_pset;
+};
 
-  double maxChi2_;
-  double nSigma_;
-  float minGoodPixelCharge_; 
-  float minGoodStripCharge_;
-  float pTChargeCutThreshold_;
+/*  need support for RefToPSet
```

Comment:
refToPSet is purely a python implementation detail and is unknown by the C++ and therefore would not be useable for validation. I can see how one might want to use it for the default generation of _cfi.py files. My suggestion for this case is to properly describe the required parameters, but do not give them values. Then have a hand written _cfi.py files which is supposed to be used.